### PR TITLE
Issue/2675

### PR DESCRIPTION
**Pull request by Arohdin**
Created actions for hiding/showing Major Moons around:
- Jupiter
- Neptune
- Saturn
- Uranus

### CODE REVIEW

File: data/assets/scene/solarsystem/planets/uranus/major_moons.asset [hunk 1]
```diff
@@ -19,7 +19,7 @@ local Miranda = {
     SegmentsPerPatch = 64,
     Layers = {}
   },
-  Tag = { "moon_solarSystem", "moon_giants", "moon_uranus", "moon_major" },
+  Tag = { "moon_solarSystem", "moon_giants", "moon_uranus", "moon_major", "moon_major_uranus"},
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/uranus/major_moons.asset:22]: Lost a space at the end

File: data/assets/scene/solarsystem/planets/uranus/major_moons.asset [hunk 2]
```diff
@@ -157,7 +157,7 @@ local Umbriel = {
...
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/uranus/major_moons.asset:160]: Same here

File: data/assets/scene/solarsystem/planets/uranus/major_moons.asset [hunk 3]
```diff
@@ -295,7 +295,7 @@ local Oberon = {
...
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/uranus/major_moons.asset:298]: Same here

File: data/assets/scene/solarsystem/planets/uranus/major_moons.asset (hunk 1 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/uranus/major_moons.asset]: Fixed in bf3fd0a20898eb76105d1878c13baf69aec5b21d

File: data/assets/scene/solarsystem/planets/uranus/major_moons.asset [hunk 4]
```diff
@@ -226,7 +226,7 @@ local Titania = {
...
```
[the remark this answers is not in the document]
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/uranus/major_moons.asset]: Fixed in bf3fd0a20898eb76105d1878c13baf69aec5b21d

File: data/assets/scene/solarsystem/planets/uranus/major_moons.asset (hunk 3 above)
Comment (Arohdin) [reply data/assets/scene/solarsystem/planets/uranus/major_moons.asset]: Fixed in bf3fd0a20898eb76105d1878c13baf69aec5b21d

**Pull request merged**